### PR TITLE
Automatically open bot on load

### PR DESCRIPTION
**Pull request by marineb**

### CODE REVIEW

File: app/Resources/views/base.html.twig [hunk 1]
```diff
@@ -459,6 +459,7 @@
         <script type="text/javascript">
             on(window, 'load', function() {
                 Kernel.boot(
+                    motionAI_Open(),
```
Comment (tgalopin) [inline app/Resources/views/base.html.twig]: If this is working, that's not normal ^^

**Comment by marineb** [comment]
@tgalopin does it look better now?

**Comment by tgalopin** [comment]
Better, but actually you should put this in https://github.com/EnMarche/en-marche.fr/blob/master/app/Resources/views/bot/_javascript.html.twig (to avoid JS issues on pages not having the bot).

**Comment by marineb** [comment]
@tgalopin sounds good! updated.

**Pull request merged**